### ISSUE TITLE
[Bug] Animated .png emotes don't render correctly

### ISSUE THREAD

**Issue by Sirs0ri** [tuskyapp/Tusky#3631]
I noticed that animated .png emotes / emojis don't render correctly in the Tusky nightlies. They either don't render at all, or they're not the size they should be. They are never animated. In the current stable Version (21) they render as they should, and animated .gif emotes are always rendered correctly, in both versions.

Some screenshots, the :ablobcatrainbow: emote (animated .png) failed to render, while :confusedlucy: (a .gif file) renders just fine. When I started taking these, the emote wouldn't show up at all, but after moving through a few pages and refreshing posts here and there I started seeing the "broken and scaled up" version of the emote. 

[link to the toot](https://corteximplant.com/@Sirs0ri/110322015569984610)
[:ablobcatrainbow: file](https://corteximplant.com/system/custom_emojis/images/000/008/957/original/615c700c048a5e4b.png)
[:confusedlucy: file](https://corteximplant.com/system/custom_emojis/images/000/011/476/original/ffc178f86e98ac20.gif)

![emote completely missing from a feed-view](https://user-images.githubusercontent.com/6765536/236628609-f94194f4-b957-402e-b3d6-b1ff26ead3cd.png)
![emote rendered broken in thread-view](https://user-images.githubusercontent.com/6765536/236628621-cdc677eb-d628-4a43-9291-57f19ef853f1.png)
![emote rendered broken on the profile view](https://user-images.githubusercontent.com/6765536/236628627-17095a85-a72a-42c2-a96d-c2144010d816.png)

Please LMK if there's any additional info you need! Thanks for looking into it.

* * * *
- Tusky Version: 
  - Test 22.0 beta 2-d2747811
- Android Version: Android 12 / OneUI 4.1
- Android Device: Samsung Galaxy s10e
- Mastodon instance (if applicable): definitely happens on corteximplant.com, but I'm not sure if it's limited to our instance.

- [x] I searched or browsed the repo’s other issues to ensure this is not a duplicate.

**Comment by connyduck** [comment]
1) We are loading all emojis in their full size, which is inefficient
2) Even if I specify the correct size, the animated emoji in question still fails to display
3) If i go even lower, I get the weird "broken and scaled up" version
4) Seems like a bug in the library?

**Comment by connyduck** [comment]
Yes, definitely a bug, downgrading to 2.23.0 fixes the problem, but upgrading to 2.25.0 does not. Will report to the maintainer. Should we downgrade in the meantime?

**Comment by Tak** [comment]
+1 for downgrade - was there a particular feature we upgraded for?

**Comment by connyduck** [comment]
Bug report: https://github.com/penfeizhou/APNG4Android/issues/188
Downgrade: https://github.com/tuskyapp/Tusky/pull/3676

> was there a particular feature we upgraded for?

No, it was auto-upgraded by Renovate
